tex/set-theory/ordinal.tex

```diff
@@ -354,7 +394,14 @@ \section{Ordinal arithmetic}
 	(Look at $\gamma \times \beta \times \alpha$.)
 \end{exercise}
 
-Exponentiation can also be so defined, though the explicit construction is less natural.
+Similar to ordinal addition, defining $\alpha \cdot (\beta+1) = (\alpha \cdot \beta) + \alpha$ makes
+sure that if $\beta < \gamma$ then $\alpha \cdot \beta < \alpha \cdot \gamma$ --- as long as $\alpha
+> 0$.
+
+Exponentiation can also be so defined, though the explicit construction is less natural ---
+since we will not use this definition in the rest of the book, you may ignore it.
```

Comment:
should we suggest the reader to ignore it, really?

For me it does caused some confusion; but at the same time, it's the natural generalization of the definition of ordinal addition and multiplication. (feel free to revert)

It's true that the concept was never used in the rest of the book though.